### PR TITLE
DASH-1116 Add modal details Board of directors.

### PR DESCRIPTION
**Pull request by trukhilio**

### CODE REVIEW

File: client/src/pages/GovernancePage/components/BoardOfDirectorsDetails/BoardOfDirectorsDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+              <a href={url} target="_blank" rel="noopener noreferrer">
+                {url}
+              </a>
+            </div>
+          ) : null}
+          <h5 className={classes.country}>Country: {country}</h5>
+          <p className={classes.text}>{text}</p>
+          <div className={classes.linksContainer}>
+            {links?.map(linkItem => {
+              const { url, logo } = linkItem;
```
Comment (andreyvEze) [inline client/src/pages/GovernancePage/components/BoardOfDirectorsDetails/BoardOfDirectorsDetails.tsx:47]:
```
links?.map(({ url, logo }) => (...))
```

File: client/src/pages/GovernancePage/components/BoardOfDirectorsDetails/BoardOfDirectorsDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  activeCandidate: CandidateProps;
+};
+
+export const BoardOfDirectorsDetails: React.FC<Props> = props => {
+  const { activeCandidate } = props;
+
+  if (!activeCandidate) return null;
+
+  const { country, id, image, lastVoteCount, links, name, status, text, url } =
+    activeCandidate || {};
```
Comment (andreyvEze) [inline client/src/pages/GovernancePage/components/BoardOfDirectorsDetails/BoardOfDirectorsDetails.tsx:20]: why do we need empty obj here?

**Review by andreyvEze — CHANGES_REQUESTED** [review]
check comments

File: client/src/pages/GovernancePage/components/BoardOfDirectorsDetails/BoardOfDirectorsDetails.tsx (hunk 1 above)
Comment (trukhilio) [reply client/src/pages/GovernancePage/components/BoardOfDirectorsDetails/BoardOfDirectorsDetails.tsx:47]: Good point

File: client/src/pages/GovernancePage/components/BoardOfDirectorsDetails/BoardOfDirectorsDetails.tsx (hunk 2 above)
Comment (trukhilio) [reply client/src/pages/GovernancePage/components/BoardOfDirectorsDetails/BoardOfDirectorsDetails.tsx:20]: Agree. Because we have check before.

**Review by andreyvEze — APPROVED** [review]

**Pull request merged**